db/migrate/20151116114008_drop_links_column_from_links_sets.rb

```diff
@@ -0,0 +1,5 @@
+class DropLinksColumnFromLinksSets < ActiveRecord::Migration
+  def change
+    remove_column :link_sets, :links
```

Comment:
It might be safer to rename this column to something like `legacy_links` until we've deployed all this, to avoid removing data accidently.